include/ofxColourspaceList.h

```diff
@@ -0,0 +1,854 @@
+#ifndef _ofxColourspaceList_h_
+#define _ofxColourspaceList_h_
+
+// Copyright OpenFX and contributors to the OpenFX project.
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @file ofxColourspaceList.h
+Contains the list of supported colourspaces.
+This file was auto-generated by scripts/genColour from studio-config-v2.1.0_aces-v1.3_ocio-v2.3.
+*/
+
+// Roles
+
+// aces_interchange
+#define kOfxColourspaceRoleAcesInterchange "aces_interchange"
+#define kOfxColourspaceRoleAcesInterchangeColourspace "ACES2065-1"
+#define kOfxColourspaceRoleAcesInterchangeMapping { "aces_interchange", "ACES2065-1" }
+
+// cie_xyz_d65_interchange
+#define kOfxColourspaceRoleCieXyzD65Interchange "cie_xyz_d65_interchange"
+#define kOfxColourspaceRoleCieXyzD65InterchangeColourspace "CIE-XYZ-D65"
+#define kOfxColourspaceRoleCieXyzD65InterchangeMapping { "cie_xyz_d65_interchange", "CIE-XYZ-D65" }
+
+// color_picking
+#define kOfxColourspaceRoleColorPicking "color_picking"
+#define kOfxColourspaceRoleColorPickingColourspace "srgb_tx"
+#define kOfxColourspaceRoleColorPickingMapping { "color_picking", "srgb_tx" }
+
+// color_timing
+#define kOfxColourspaceRoleColorTiming "color_timing"
+#define kOfxColourspaceRoleColorTimingColourspace "ACEScct"
+#define kOfxColourspaceRoleColorTimingMapping { "color_timing", "ACEScct" }
+
+// compositing_log
+#define kOfxColourspaceRoleCompositingLog "compositing_log"
+#define kOfxColourspaceRoleCompositingLogColourspace "ACEScct"
+#define kOfxColourspaceRoleCompositingLogMapping { "compositing_log", "ACEScct" }
+
+// data
+#define kOfxColourspaceRoleData "data"
+#define kOfxColourspaceRoleDataColourspace "Raw"
+#define kOfxColourspaceRoleDataMapping { "data", "Raw" }
+
+// matte_paint
+#define kOfxColourspaceRoleMattePaint "matte_paint"
+#define kOfxColourspaceRoleMattePaintColourspace "ACEScct"
+#define kOfxColourspaceRoleMattePaintMapping { "matte_paint", "ACEScct" }
+
+// scene_linear
+#define kOfxColourspaceRoleSceneLinear "scene_linear"
+#define kOfxColourspaceRoleSceneLinearColourspace "ACEScg"
+#define kOfxColourspaceRoleSceneLinearMapping { "scene_linear", "ACEScg" }
```

Comment:
I've removed the role mapping info and added docs to explain that roles should be used where the plug-in wants to specify a general type of colourspace rather than a specific one.